### PR TITLE
added ZipFile instead of using raw p7zip_jll, still not completely re…

### PR DESCRIPTION
**Pull request by jvieyras**
…moved.

All tests pass now.

### CODE REVIEW

File: src/write.jl [hunk 1]
```diff
@@ -139,33 +134,29 @@ function Base.write(
         end
     end
 
-    origin = pwd()
     try
-        mktempdir() do tmpdir
-            cd(tmpdir)
+        tmpdir = mktempdir()
+        cd(tmpdir) do
```
Comment (matthijscox-asml) [inline src/write.jl:139]: oh nice :)

File: Project.toml [hunk 2]
```diff
@@ -11,6 +11,7 @@ Images = "916415d5-f1e6-5110-898d-aaa5f9f070e0"
 Pkg = "44cfe95a-1eb2-52ea-b672-e2afdf69b78f"
```
Comment (matthijscox-asml) [inline Project.toml:11]: Pkg can be removed then?

Comment (jvieyras) [reply Project.toml:11]: haven't fully removed `p7zip_jll` usage yet, just the minimum to make the tests pass.

File: src/PPTX.jl [hunk 3]
```diff
@@ -2,6 +2,7 @@ module PPTX
 
 using XMLDict
 using EzXML
+using ZipFile
 using DataStructures
 
 import Tables
```
Comment (matthijscox-asml) [inline src/PPTX.jl:8]: At line 11, we can remove `import Pkg.PlatformEngines: exe7z` ?

Comment (jvieyras) [reply src/PPTX.jl:8]: with `ZipFile` though, you don't need to write files to disk, but would need to rethink a bit more of the structure of the code.

Comment (matthijscox-asml) [reply src/PPTX.jl:8]: let's leave this idea for another version. We could leave this PR open for now, or open an issue instead.

**Comment by matthijscox-asml** [comment]
We switched to ZipArchives.jl in https://github.com/ASML-Labs/PPTX.jl/pull/31, so closing this PR

**Pull request closed without merging**